physical/oci/oci.go

```diff
@@ -63,6 +63,7 @@ var (
 
 type Backend struct {
 	client         *objectstorage.ObjectStorageClient
+	region         string
```

Comment:
Region doesn't seem to be used outside of its constructor, does it have to be stored?

Reply:
I'll defer to your expertise here.  I was mostly following the patterns of the other variables taken from the config but when I think about it I would imagine you'd want all information about the Backend to be captured?  If it causes bloat/harm let me know but I can see valid use-cases when implementing better HA features or KMS features in the future for OCI.

Reply:
Sorry for the delayed response, I missed the reply notification on my inbox! The other fields are stored because they end up being referenced elsewhere outside of `NewBackend`. For instance, `Backend.bucketName` is used on the backend's  `Get`, `Put`, `List`, and `Delete` methods.

Reply:
I'll be merging this and taking care of Calvin's comment in a separate PR.

Reply:
https://github.com/hashicorp/vault/pull/10248